recipes-app/node-red/node-red_2.2.3.bb

```diff
@@ -10,13 +10,17 @@
 #
 
 inherit npm
+inherit node-red-user
```

Comment:
Node-RED is not an updatable package, is it? Why now suddenly?

Reply:
`node-red-user` does not mean it is an updatable package. It just means that this package installs / prepares something in the node-red user directory. In this case, this is the systemd service that needs to execute node-red from the user directory.

Reply:
The node-red-user directory is supposed to be updatable, that's its whole purpose. Otherwise, you would install it into the node-red modules dir, just like node-red-gpio.

Reply:
Moved logic to inc file to avoid confusion about naming scheme